### ISSUE TITLE
Music Page

### ISSUE THREAD

**Issue by nickejanssen** [nickejanssen/WOM#1]
Summary:
Build music page so that record holders, and general users can listen to them music, download the album/stems. 

We will be using:
- HTML5 Vinyl-styled Music Player:
  * http://codecanyon.net/item/html5-vinylstyled-music-player/5435457
  * [codecanyon-5435457-html5-vinylstyled-music-player.zip (https://github.com/nickejanssen/WOM/files/156032/codecanyon-5435457-html5-vinylstyled-music-player.zip)
- AWS S3 to store the music

Tasks:
- [x] Configure application with AWS
 - [x] http://docs.aws.amazon.com/sdk-for-ruby/latest/DeveloperGuide/aws-ruby-sdk-getting-started.html#aws-ruby-sdk-configuration
- [x] Store music in S3 Bucket
- [x] Set up Music Player
 - [x] Set up the HTML Structure
 - [x] Set up the JS to add the songs

**Comment by PAMulligan** [comment]
I've configured the application, although I don't know if you're storing your secrets in an .env file or not. Obviously, I can't access your S3 account, so I'm going to move ahead to setting up the music player.

**Comment by nickejanssen** [comment]
I was, but I took them out when I was having issues. I'll add them again so you will have access. I attempted to install the plugin, but was only getting the background of the player. I double checked through the css and html that I was referencing everything correctly and it looked fine. Might be that I didnt add the assets in the pipeline appropriately. Let me know if you figure it out. 

Thanks again for contributing, it makes a huge difference.r

**Comment by nickejanssen** [comment]
I added the keys to the environment. They can be accessed using: 

ENV["ACCESS_KEY_ID"]
ENV["SECRET_ACCESS_KEY"]
ENV["BUCKET"]

**Comment by PAMulligan** [comment]
Can you add your region in the .env file under AWS_REGION?

**Comment by nickejanssen** [comment]
I added it. You should be good to go. Let me know if you have any issues.

**Comment by PAMulligan** [comment]
I don't think I can access the S3 bucket without the .env file being part of the repo. I have to set the region manually, and while I haven't tried to implement video files from the bucket yet, my thought is that I won't be able to login. I think the simplest workaround is just to use my own bucket, so I'm going to take this test mp4 and upload it to my AWS.

I have the player loaded on the page, but when I try to play the video, the link appears to be broken. Is this the issue you were running into?

**Comment by nickejanssen** [comment]
It should only be audio files that are needed. You might be using the womAnimation.mp4 file  that I have in the assets. I added a couple songs to assets/audio that you can use, 01_Raise_Your_Voices.mp3 and 02_Word_of_Mouth.mp3. The issue I was running into, was that none of the components of the music player would load except for the background. If you got to the music page, you can see what I mean.

**Comment by PAMulligan** [comment]
I solved that issue by changing the class to 'quickPlate' instead of
'demoPlate'. So, the controls are showing up, but it's not processing the
video file correctly. I'm not really sure why, it's called properly in the
asset pipeline. I'll see if I have time tonight to try out those audio
files.

**Comment by nickejanssen** [comment]
Great, I added the audio files in the public folder, made some changes to plate.js, and now it is playing the first song in playlist.js correctly. The playlist in the music player isnt generating correctly though. It should also play the second song as well, but isnt. I think its just a matter of adjusting the js. 

I also edited the application.js required fields. 

Great work by the way! Thanks again.
